src/components/SummaryManagement/SummaryReceiverPopup.jsx

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable react/prop-types */
+import React, { useState, useEffect } from 'react';
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter, Container, Alert } from 'reactstrap';
+import MembersAutoComplete from './MembersAutoComplete';
+//import hasPermission from 'utils/permissions';
+
+const TeamMembersPopup = React.memo(props => {
+  // debugger;
+  const closePopup = () => {
+    props.onClose();
+  };
+  const [selectedUser, onSelectUser] = useState(undefined);
+  const [isValidUser, onValidation] = useState(true);
+  const [searchText, setSearchText] = useState('');
+
+  const onAddUser = () => {
+    cd;
+    if (
+      selectedUser &&
+      (!props.members ||
+        !props.members.teamMembers ||
+        !props.members.teamMembers.some(x => x._id === selectedUser._id))
```

Comment:
```suggestion
        !props.members.summaryReceivers  ||
        !props.members.summaryReceivers .some(x => x._id === selectedUser._id))
```
Could add duplicate receivers

Reply:
It would probably be better if you could just abstract the TeamMemberPopup to include both.

Reply:
It has been revised; thanks for the advice and for helping me debug it; appreciate it!